crates/pixi_consts/src/consts.rs

```diff
@@ -8,9 +8,9 @@ pub use pixi_manifest::consts::*;
 pub const PROJECT_MANIFEST: &str = "pixi.toml";
```

Comment:
Q: Is this the best way of using a package or should these consts be moved into their respective crates. Like manifest information in `pixi_manifest` and config stuff in `pixi_config`?
Not holding back this pr, just interested in the future design.

Reply:
Yeah I tried that for this one, but ran into circular references. Should be something we think about in the future indeed.